frontend/app/src/components/home/AccountInfo.svelte

```diff
@@ -4,16 +4,23 @@
     import { toastStore } from "../../stores/toast";
     import ContentCopy from "svelte-material-icons/ContentCopy.svelte";
     import { iconSize } from "../../stores/iconSize";
-    import type { CreatedUser } from "openchat-client";
+    import type { CreatedUser, OpenChat } from "openchat-client";
     import { copyToClipboard } from "../../utils/urls";
+    import { getContext } from "svelte";
+    import { ICP_SYMBOL } from "openchat-shared";
```

Comment:
I'm not sure it really matters but I don't think we generally import from openchat-shared in the app project. We always import from the openchat-client (which re-exports anything needed from shared). This would be nicer if / when we wanted to let third parties use openchat-client.